apps/frontend/components/edit-namespace.tsx

```
@@ -209,6 +228,40 @@ export function EditNamespace({
               )}
             </div>
 
+            {/* Namespace Ownership */}
+            <div className="space-y-2">
+              <label className="text-sm font-medium">
+                {t("namespaces:ownership")}
+              </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)